README.rst

```diff
@@ -0,0 +1,162 @@
+If you use:                              then type:
+---------------------------------------- ------------------------------------------
+a bash-compatible shell (bash/zsh)       ``source util/quickstart/setchplenv.bash``
+a csh-compatible shell (csh/tcsh)        ``source util/quickstart/setchplenv.csh``
+the fish shell (fish)                    ``. util/quickstart/setchplenv.fish``
+the Bourne shell (sh)                    ``. util/quickstart/setchplenv.sh``
+======================================== ==========================================
+
+
+You must be in the top-level Chapel directory for these command
+scripts to work properly.  Note that there is no requirement to use
+these scripts long-term, they are merely designed to get new users
+up and running quickly.  See ``doc/chplenv.rst`` for a complete
+description of Chapel's environment variables and their options.
+
+If you use a shell other than the ones listed above and are able to
+create an equivalent ``util/quickstart/setchplenv.*`` script for it,
+please consider contributing a copy of it back to the project for
+inclusion in the next release.
+
+
+3) Build the compiler and runtime libraries using:
+
+        ``gmake``
+
+   or if your default make is GNU make compatible (as on Mac OS X or
+   Cygwin), use:
+
+        ``make``
+
+   Parallel build execution (e.g. ``gmake -j``) is supported.
+
+4) ``csh``/``tcsh`` users only: Update your shell's path cache using:
+
+        ``rehash``
+
+
+5) To ensure you have installed Chapel properly, you can optionally run an
+   automatic sanity check using a few example programs. For this to work
+   correctly, you will need python, and ``python-devel`` packages installed on your
+   system. See ``doc/prereqs.rst`` for more information.
+
+        ``gmake check``
+
+   or:
+
+        ``make check``
+
+
+6) Compile an example program using:
+
+        ``chpl -o hello examples/hello.chpl``
+
+
+7) Execute the resulting executable:
+
+       ``./hello``
+
+
+8) Experiment with Chapel in Quick Start mode to your heart's content.
+
+   Once you are comfortable with Chapel and interested in using a
+   full-featured version in the preferred configuration:
+
+   a) Open up a new shell to avoid inheriting the previous environment
+      settings.
+
+   b) Repeat steps 1-7 above, but in Step 2, use ``util/setchplenv.*``
+      instead of ``util/quickstart/setchplenv.*``
+
+   This will set up your environment to use Chapel in the preferred
+   configuration.  Making this configuration will involve building one
+   or more third-party packages which will increase the overall build
+   time.  If you run into any portability issues, please let us know
+   at chapel_info@cray.com.
+
+
+9) If you plan to do performance studies of Chapel programs, be sure
+   to (a) use the full-featured version and (b) read the ``PERFORMANCE``
+   file in this directory to avoid common pitfalls.
+
```

Comment:
I've been thinking for awhile now that we should add a 10th bullet to this list along these lines:

10) All of the instructions above describe how to run Chapel programs in a single-locale / shared-memory mode.  To run on multiple compute nodes, please refer to README.multilocale.